packages/frontend/src/components/Election/Results/ViewElectionResults.tsx

```diff
@@ -43,7 +61,26 @@ const ViewElectionResults = () => {
             <Typography variant="h4" component="h4">
               {t("results.election_title", { title: election.title })}
             </Typography>
-            {isPending && <div> {t("results.loading_election")} </div>}
+
+
+            {/* Voting method and learning link */}
+              {/* Test of array of voting methods and learning links */}
+              {Array.from({ length: raceCount }).map((_, i) => (
+                  <Typography key={i} variant="h5" component="h5">
```

Comment:
Thanks for adding this! I would recommend moving the voting method text to the Results file instead https://github.com/Equal-Vote/bettervoting/blob/main/packages/frontend/src/components/Election/Results/Results.tsx#L488

I think that'll also be a little easier since you'll already have the specific race handy in that context

Also when it's updated, can you post a screenshot in the PR description?

Reply:
@ArendPeter I tried doing that, but I could not get the voting method and learning link to appear in the correct position, before the race results.  The data is retrieved asynchronously, and that does not finish until the race results are rendered.

Reply:
Ah ok, perhaps we can co-work on that then. Feel free to book a meeting with me at starvoting.org/meeting 🙂